### DISCUSSION TITLE
How to access OpenMPTCProuter via VPN?

### DISCUSSION THREAD

**Discussion by julianguttzeit** [Ysurac/openmptcprouter#2130] [Q&A]
Hi Guys,

I just want to aks if there is a good solution for connecting to my OpenMPTCProuter via VPN?
The case is: I build a video streaming suitcase with 3 LTE connections and want to control the whole Infrastructure (ATEM Device, Raspberry Pi's, etc.) remotely to support my employees in case of a technical Issue.

Also I use the OpenMPTCProuter in the office and want to connect remotely. Is it possible via the connected FritBox (it takes a WAN out of the OpenMPTCProuter and gets a local IP right now)? Or possible on the OpenMPTCProuter? Bypassing the Bonding to a VDLS WAN or something else? Or do I need to do some Server changes for beeing able to access the OpenMPTCProuter without losing the bonding?

Thank you very much!

Best
Julian

**Comment by gre2gor** [comment]
It's possible. Network > Firewall > add ports to access different connected gear. What do you use for streaming? I also use openmptcp with great success.
